storybookjs/storybook
# upgrade vitejs and esbuild

Issue: #20190 

## What I did

This upgrades the dependencies so that a fresh pull will compile and start. If you have an existing build please clear out all files NOT checked in to the code base (git ignored, including node_modules)

## How to test

- Test yarn start and yarn ci-tests

If your answer is yes to any of these, please make sure to include it in your PR.
